discord/ext/modal_paginator/core.py

```diff
@@ -490,6 +492,15 @@ def from_text_inputs(
                     default_title="Please answer the following questions",
                     # other parameters
                 )
+        max_inputs_per_modal: :class:`int`
+            The maximum inputs to add per each modal created. Must be between 1 and 5,
+            both included. Defaults to ``5``.
+
+            .. versionadded:: 1.3
+        titles_step: :class:`int`
+            How much indexes should the modal increase per each modal created.
```

Comment:
Missing default value.
```suggestion
            How much indexes should the modal increase per each modal created.
            Defaults to ``1``.
```